lib/sender.js

```diff
@@ -115,11 +115,15 @@ function send(client) {
 
             request(req, (err, response, body) => {
                 if (err) {
-                    return reject(`Problem with Request: ${err.message}`);
+                    return reject(
+                        new Error(`Problem with UPS Request: ${err.message}`)
```

Comment:
Typically I use VError in this scenario to do this:

```js
return reject(
  new VError(err, 'request to UPS encountered an error')
)
```

VError will retain original error context, and append our message which is very nice. I did not do it here since it introduces a new dependency which you may not desire.